examples/IrisClassificationTorchScript/iris_classification.py

```diff
@@ -83,21 +83,7 @@ def configure_optimizers(self):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-
-    parser.add_argument(
```

Comment:
Don't we still need similar defaults with the upgrade?

Reply:
No @chauhang . `parser = pl.Trainer.add_argparse_args(parent_parser=parser)` adds all the PTL arguments to the parser. If we keep the same arguments here, the following error will be thrown

```
Traceback (most recent call last):
  File "iris_classification.py", line 197, in <module>
    parser = pl.Trainer.add_argparse_args(parent_parser=parser)
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/pytorch_lightning/trainer/trainer.py", line 2461, in add_argparse_args
    return add_argparse_args(cls, parent_parser, **kwargs)
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/pytorch_lightning/utilities/argparse.py", line 268, in add_argparse_args
    parser.add_argument(
  File "/home/ubuntu/anaconda3/lib/python3.8/argparse.py", line 1386, in add_argument
    return self._add_action(action)
  File "/home/ubuntu/anaconda3/lib/python3.8/argparse.py", line 1590, in _add_action
    action = super(_ArgumentGroup, self)._add_action(action)
  File "/home/ubuntu/anaconda3/lib/python3.8/argparse.py", line 1400, in _add_action
    self._check_conflict(action)
  File "/home/ubuntu/anaconda3/lib/python3.8/argparse.py", line 1539, in _check_conflict
    conflict_handler(action, confl_optionals)
  File "/home/ubuntu/anaconda3/lib/python3.8/argparse.py", line 1548, in _handle_conflict_error
    raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument --max_epochs: conflicting option string: --max_epochs
```